build.gradle

```
@@ -56,4 +68,24 @@ test {
 
 jacocoTestReport {
 	dependsOn test
+}
+
+// spotless
+compileJava {
+	dependsOn spotlessApply
+}
+
+spotless {
+  java {
+    // Use the default importOrder configuration
+    importOrder()
+    // optional: instead of specifying import groups directly you can specify a config file
+    // export config file: https://github.com/diffplug/spotless/blob/main/ECLIPSE_SCREENSHOTS.md#creating-spotlessimportorder
+
+    removeUnusedImports()
+
+    googleJavaFormat() // has its own section below
```

Remove comment copied from Spotless docs
```suggestion
    googleJavaFormat()
```